roles/cleanup/tasks/logging.yml

```diff
@@ -1,10 +1,19 @@
-- name: keystone logging.conf
-  file: dest=/etc/keystone/logging.conf state=absent
-- file: dest=/var/log/keystone/error.log state=absent
-- file: dest=/var/log/keystone/access.log state=absent
+- name: remove old log configs / files
+  file: dest={{ item }} state=absent
+  with_items:
+    - /etc/nova/logging.conf
+    - /etc/cinder/logging.conf
+    - /etc/keystone/logging.conf
```

Comment:
See my [comment](https://github.com/blueboxgroup/ursula/pull/781#issuecomment-72742600) with links to the HEAD of RELEASE-1.0.x missing any references to `logging.conf`